bandwidth/bandwidth-impl/build.gradle

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+dependencies {
+    implementation project(path: ':bandwidth-store')
+    api project(path: ':bandwidth-store')
+    implementation project(path: ':common')
+    implementation project(path: ':di')
+    implementation project(path: ':browser-api')
+    implementation project(path: ':statistics')
+    implementation project(path: ':vpn')
```

Comment:
With the comment [here](https://github.com/duckduckgo/Android/pull/1866/files#r852033158) we probably can remove this dependency in favor of `:vpn-api` (?)